pingserver/main.go

```diff
@@ -15,19 +15,43 @@
 package main
 
 import (
-	"fmt"
 	"log"
 	"net/http"
 	"os"
 )
 
+// greetHandler handles the index page.
```

Comment:
We are not using the pingserver yet. Please file issues before contributing major changes.

Reply:
I thought the blackbox endpoint was using it. I'll close the PR, then, and if it's still relevant in the future, I'll reopen it.